plugins/Deflect/DeflectPlugin.cpp

```diff
@@ -81,31 +81,32 @@ class DeflectPlugin::Impl
                 _closeStream();
         }
 
-        if (_previousHost != _params.getHostname())
-        {
-            _params.setEnabled(true);
-            _previousHost = _params.getHostname();
-        }
-
         const bool deflectEnabled = _params.getEnabled();
         if (_stream && _stream->isConnected() && !deflectEnabled)
             _closeStream();
 
-        const bool observerOnly = _engine.haveDeflectPixelOp();
+        const bool observerOnly = _params.usePixelOp();
         if (deflectEnabled && !_stream && _startStream(observerOnly))
```

Comment:
observerOnly parameter not needed as _startStream has access to _params (and in facts uses them)